src/components/BaseLayerSwitcher/BaseLayerSwitcher.scss

```diff
@@ -0,0 +1,74 @@
+.rs-base-layer-switcher {
+  position: relative;
+  height: 80px;
+  width: 115px;
+  transition: 250ms width;
+  overflow: hidden;
+
+  &.open {
+    width: 360px;
+  }
+
+  .rs-base-layer-switch-layers {
```

Comment:
Don't put child classes to this class if it's not necessary. it wil be easier to overide if we need it .
Forexamples:  
```
.rs-base-layer-switcher { ...
   .rs-base-layer-switch-layers {...}
   .rs-base-layer-switch-button {...}
   .rs-base-layer-switch-title {...}
}

Reply:
Yes that makes a lot of sense